Formula/freeimage.rb

```diff
@@ -25,6 +25,13 @@ class Freeimage < Formula
   end
 
   def install
+    # Temporary workaround for ARM. Upstream tracking issue:
+    # https://sourceforge.net/p/freeimage/bugs/325/
+    # https://sourceforge.net/p/freeimage/discussion/36111/thread/cc4cd71c6e/
+    if Hardware::CPU.arm?
+      ENV.append "CFLAGS", "-DPNG_ARM_NEON_OPT=0"
+      ENV.append "CXXFLAGS", "-DPNG_ARM_NEON_OPT=0"
```

Comment:
There is no need to override CXXFLAGS since libpng is plain C.

Reply:
You are right.
I will remove CXXFLAGS.
Thank you!